Ginger/Ginger/ALM/ALMItemsFieldsConfigurationPage.xaml.cs

```diff
@@ -92,16 +108,36 @@ public void ShowAsWindow(bool refreshFields = true, eWindowShowStyle windowStyle
 
         private void Save(object sender, RoutedEventArgs e)
         {
-            // TODO Rearrange save function to keep old fields value.
-            ObservableList<ExternalItemFieldBase> tempItemList = ALMIntegration.Instance.GetUpdatedFields(mItemsFields, false);
-            if (tempItemList.Any())
+            if (mAlmConfigType.Equals(eALMConfigType.MainMenu))
+            {
+                // TODO Rearrange save function to keep old fields value.
+                ObservableList<ExternalItemFieldBase> tempItemList = ALMIntegration.Instance.GetUpdatedFields(mItemsFields, false);
+                if (tempItemList.Any())
+                {
+                    WorkSpace.Instance.Solution.ExternalItemsFields = tempItemList;
+                }
+                WorkSpace.Instance.Solution.SolutionOperations.SaveSolution(true, SolutionGeneral.Solution.eSolutionItemToSave.ALMSettings);
+            }
+            else
             {
-                WorkSpace.Instance.Solution.ExternalItemsFields = tempItemList;
+                RunSetConfig runSetConfig = WorkSpace.Instance.RunsetExecutor.RunSetConfig;
```

Comment:
No need to do Save for Operation because it will save all Run set.
Need to hide Save button and replace it with "Ok" button

Reply:
ok